app/assets/stylesheets/patternfly_and_overrides.scss

```diff
@@ -321,7 +321,7 @@ a.btn-info span {
   }
 }
 
-.dropdown-submenu > .dropdown-menu {
+.dropdown-menu {
```

Comment:
The `dropdown-menu` class is present in multiple other locations in the app. To make sure this only affects the one you want, you should use `.org-switcher > .dropdown-menu`.

Reply:
wouldn't we like to add a max-height + scrollbar to a general drop down menu?

Reply:
not if it will break existing pages - did you check that other pages using this class don't break?